macOS/DuckDuckGo/Updates/UpdateController.swift

```diff
@@ -186,12 +231,44 @@ final class UpdateController: NSObject, UpdateControllerProtocol {
 
         Logger.updates.log("Checking for updates respecting rollout")
 
+        userDriver?.userCheckedForUpdates()
         updater.checkForUpdatesInBackground()
     }
 
+    private var isBuildExpired: Bool {
+        canBuildsExpire && shouldForceUpdateCheck
+    }
+
+    @discardableResult
+    private func discardCurrentUpdateIfExpiredAndCheckAgain() -> Bool {
+        guard isBuildExpired else {
+            return false
+        }
+
+        userDriver?.cancelAndDismissCurrentUpdate()
+        updater = nil
+
+        DispatchQueue.main.asyncAfter(deadline: .now() + 0.1) { [weak self] in
+            guard let self,
+                  let updater = try? configureUpdater(needsUpdateCheck: true) else {
+                return
+            }
+            self.updater = updater
+
+            userDriver?.userCheckedForUpdates()
+            updater.checkForUpdates()
```

Comment:
Similar to a previous comment, this will bypass our rollout group, which likely is not what we want.

Reply:
Never mind.  I changed the logic so we won't skip rollout now.